### PR TITLE
Feature: Add additional integration GraphQL fns

### PR DESCRIPTION
**Pull request by chasen-bettinger**
- Moved generated.go outside of the jupiterone package and into an internal package
- Added genqlient files so that we can generate additional graphql functions
- Refactored integration code to use genqlient graphql functions
- Added ability to get an integration instance and update an integration instance

### CODE REVIEW

File: examples/update-integration-instance/example.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+		input := graphql.UpdateIntegrationInstanceInput{
+			Name:                        integrationInstance.IntegrationInstance.Name,
+			SourceIntegrationInstanceId: integrationInstance.IntegrationInstance.SourceIntegrationInstanceId,
+			PollingInterval:             integrationInstance.IntegrationInstance.PollingInterval,
+			PollingIntervalCronExpression: graphql.IntegrationPollingIntervalCronExpressionInput{
+				DayOfWeek: integrationInstance.IntegrationInstance.PollingIntervalCronExpression.DayOfWeek,
+				Hour:      integrationInstance.IntegrationInstance.PollingIntervalCronExpression.Hour,
+			},
+			Description:     integrationInstance.IntegrationInstance.Name,
```
Comment (zemberdotnet) [inline examples/update-integration-instance/example.go]:
```suggestion
			Description:     integrationInstance.IntegrationInstance.Description,
```

File: internal/graphql/generated.go [hunk 2]
```diff
@@ -0,0 +1,2413 @@
...
+
+type UpdateIntegrationInstanceInput struct {
+	Name                          string                                        `json:"name"`
+	SourceIntegrationInstanceId   string                                        `json:"sourceIntegrationInstanceId"`
+	PollingInterval               IntegrationPollingInterval                    `json:"pollingInterval"`
+	PollingIntervalCronExpression IntegrationPollingIntervalCronExpressionInput `json:"pollingIntervalCronExpression"`
+	Description                   string                                        `json:"description"`
+	Config                        map[string]interface{}                        `json:"config"`
+	OffsiteComplete               bool                                          `json:"offsiteComplete"`
+}
```
Comment (zemberdotnet) [inline internal/graphql/generated.go]: It doesn't look like the generated code is going to handle optional fields very well. Not all integrations are supposed/want to have `PollingIntervalCronExpressions` and `SourceIntegrationInstanceId`. I'm not sure the API will smoothly handle the empty values Go will serialize.

File: jupiterone/integrations.go [hunk 3]
```diff
@@ -108,167 +17,33 @@ type DeleteInstanceResult struct {
...
-	}
-
-	return resp.IntegrationDefinition, nil
+func (s *IntegrationService) GetDefinition(id string) (*graphql.GetIntegrationDefinitionResponse, error) {
+	return graphql.GetIntegrationDefinition(context.Background(), s.client.gqlClient, id)
 }
 
 // ListInstances list the integration instances for the JupiterOne account.
-// ListInstances returns a reference to an IntegrationInstanceResponse which
+// ListInstances returns a reference to an AnIntegrationInstanceResponse which
```
Comment (zemberdotnet) [inline jupiterone/integrations.go]:
```suggestion
// ListInstances returns a reference to an IntegrationInstanceResponse which
```

File: internal/graphql/generated.go (hunk 2 above)
Comment (chasen-bettinger) [reply internal/graphql/generated.go]: @zemberdotnet  Great catch! Thanks for looking over my shoulder. Take a peek now, I think it's closer to what we want.

File: internal/graphql/generated.go [hunk 4]
```diff
@@ -0,0 +1,2413 @@
...
+type CreateIntegrationInstanceInput struct {
+	Name                          string                                        `json:"name"`
+	SourceIntegrationInstanceId   string                                        `json:"sourceIntegrationInstanceId"`
+	PollingInterval               IntegrationPollingInterval                    `json:"pollingInterval"`
+	PollingIntervalCronExpression IntegrationPollingIntervalCronExpressionInput `json:"pollingIntervalCronExpression"`
+	IntegrationDefinitionId       string                                        `json:"integrationDefinitionId"`
+	Description                   string                                        `json:"description"`
+	Config                        map[string]interface{}                        `json:"config"`
+	OffsiteComplete               bool                                          `json:"offsiteComplete"`
+}
```
Comment (zemberdotnet) [inline internal/graphql/generated.go]: Might be good to just do the same thing here.

Comment (chasen-bettinger) [reply internal/graphql/generated.go]: @zemberdotnet done!

**Review by zemberdotnet — APPROVED** [review]

**Pull request merged**